flight_data_collect/drone_communication/mavlink_utils.py

```diff
@@ -34,15 +34,18 @@ def get_mavlink_messages(connect_address):
             msg = _get_mavlink_message(mavlink, msg_type, connect_address)
```

Comment:
are we supposed to still be using USEFUL_MESSAGES to decide which messages to receive here?